app/views/docs/oauth-providers/spotify.md

```diff
@@ -36,7 +36,7 @@ const account = new Account(client);
 
 client
     .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
-    .setProject('[PROJECT_ID]') // Your project ID
+    .setProject('[PROJECT_ID]')                         // Your Project ID
```

Comment:
The comment still doesn't look aligned.  Can you please recheck? @asutoshranjan

Reply:
Surely! checking it